problem_builder/tests/unit/test_problem_builder.py

```diff
@@ -1,9 +1,15 @@
-import unittest
 import ddt
+import unittest
+
 from mock import MagicMock, Mock, patch
+from random import random
+
 from xblock.field_data import DictFieldData
+
 from problem_builder.mcq import MCQBlock
-from problem_builder.mentoring import MentoringBlock, MentoringMessageBlock, _default_theme_config
+from problem_builder.mentoring import (
+    MentoringBlock, MentoringMessageBlock, _default_theme_config, _default_options_config
```

Comment:
`_default_theme_config` doesn't seem to be used - also that should have caused a pylint error, shouldn't it have?